### ISSUE TITLE
Example does not compile

### ISSUE THREAD

**Issue by jreznot** [jdi-templates/jdi-light-junit-template#3]
```
Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project jdi-light-junit-template: Compilation failure
[ERROR] /home/artamonov/test/qa/jdi-light-junit-template/src/main/java/org/mytests/uiobjects/example/site/custom/MultiDropdown.java:[26,47] type argument com.epam.jdi.light.asserts.generic.UISelectAssert is not within bounds of type-variable A
```

**Comment by clockworkbanana** [comment]
To be fixed in scope of: https://github.com/jdi-testing/jdi-light/issues/2099

**Comment by clockworkbanana** [comment]
Fixed in scope of jdi-testing/jdi-light#2099;
the changes were successfully merged into master branch.
